### ISSUE TITLE
U12 has genre

### ISSUE THREAD

**Issue by pasqLisena** [DOREMUS-ANR/itema3converter#16]
Revoir la conversion de F22 U12 has genre M5 Genre.

Il faudrait exploiter l’alignement entre les genres Item3 (type_musical) et ceux de iaml et mettre les genres alignés et seulement eux ici, sans les enlever des catégorisations

-- issue by _Martine_

**Comment by pasqLisena** [comment]
I do not get this. If you say that some "itema3-genres" are not really genres, the more clean solution is to remove them from the vocabulary. Then, the rule would be: they are genres if they are in the vocabulary.
 
The workaround you proposed is expensive in coding time.

**Comment by rtroncy** [comment]
I agree, this is a "won't fix" for me too. The idea is that we always retain the source data, so we use the ITEMA3 genre url, which may be aligned to other genre controlled vocabularies, but then, it is retrievable using the alignments.

@pasqLisena In order to get all all F22 that has a U12_has_genre a particular genre, e.g. `<http://data.doremus.org/vocabulary/itema3/genre/24>`, do you need to add in the query `OR ?x skos:exactMatch <http://data.doremus.org/vocabulary/itema3/genre/24>` or does Virtuoso o the inference itself?

**Comment by pasqLisena** [comment]
The inference is done with `owl:sameAs` but not with `skos:exactMatch`.

I currently do `?expression mus:U12_has_genre / skos:exactMatch* < http://data.doremus.org/vocabulary/itema3/genre/24 >` for retrieving everything.
